.gitlab-ci.yml

```diff
@@ -1,7 +1,7 @@
 ---
 generate-pipeline:
   stage: build
-  image: python:3.9-slim
+  image: thomass/gcip
```

Comment:
- I would like to not create a cyclic dependency here ("Python Project gcip" -> "Docker Image gcip" -> "Python Project gcip" -> ...)
- Even if we do, the way to use the Docker image looks like following:
  https://dbsystel.github.io/gitlab-ci-python-library/user/index.html#_configuring_your_project_to_use_gcip
- Using a "basic" python image prevents concerns about API breaks within our own project. The input (new gcip code) should not depend on the output (old gcip Docker image from old code).